Core/RoverControllerPresets/DriveControllers/DirectDriveController.cs

```diff
@@ -6,6 +6,9 @@ namespace RoverControlApp.Core.RoverControllerPresets.DriveControllers;
 
 public class DirectDriveController : IRoverDriveController
 {
+	public float SpeedModifier => LocalSettings.Singleton.SpeedLimiter.Enabled ? LocalSettings.Singleton.SpeedLimiter.MaxSpeed : 1f;
```

Comment:
This can be marked static
```suggestion
	public static float SpeedModifier => LocalSettings.Singleton.SpeedLimiter.Enabled ? LocalSettings.Singleton.SpeedLimiter.MaxSpeed : 1f;
```

Reply:
Adressed in [ea00d3e](https://github.com/wisniax/RoverControlApp/pull/36/commits/ea00d3e6c6a7ac37d62a2149d24415175ee49ced)